src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -46,7 +46,14 @@ const MostReadSection = styled.section.attrs(() => ({
   role: 'region',
   'aria-labelledby': 'Most-Read',
   'data-e2e': 'most-read',
-}))``;
+}))`
```

Comment:
Does this change affect most read on article pages?

Reply:
That's a good point. Those styles are also applied when on Article Pages, making the section to go out the viewport due to the `width:100%`

<img width="1017" alt="Screenshot 2020-03-03 at 11 59 00" src="https://user-images.githubusercontent.com/46446236/75773839-95c48680-5d46-11ea-8936-aac8db8dc51c.png">

Reply:
Looking into this

Reply:
@tochwill the most-read component overrides this styling on article pages.

**Most read on article pages.**
![Screen Shot 2020-03-03 at 3 13 02 PM](https://user-images.githubusercontent.com/21374761/75774718-93bbf100-5d61-11ea-93d5-2abc3ad6b1d4.png)

**Most read on front-pages.** (_constrained to 63rem at widths above 1280px_)
![Screen Shot 2020-03-03 at 3 12 22 PM](https://user-images.githubusercontent.com/21374761/75774730-974f7800-5d61-11ea-9c80-10f8e7e80d2b.png)

Reply:
Good catch.
This should be fixed now